src/test/robotframework/possibilityToLogIn/validLogin.txt

```diff
@@ -0,0 +1,13 @@
+*** Settings ***
+Documentation     A test suite with a single test for valid login.
+Resource          resource.txt
+
+*** Test Cases ***
+Valid Login
```

Comment:
Could you please try to create report:
[http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#report-file]
?